third_party/2and3/pymysql/cursors.pyi

```diff
@@ -1,7 +1,9 @@
-from typing import Union, Tuple, Any, Dict, Optional, Text, Iterator, List
+from typing import Any, Dict, Iterator, List, Optional, Text, Tuple, TypeVar, Union
+
 from .connections import Connection
 
 Gen = Union[Tuple[Any, ...], Dict[str, Any]]
+_SelfT = TypeVar("_SelfT", bound="Cursor")
```

Comment:
The `bound` is actually unnecessary, because it's already implied by using this for a certain `self` argument.